VagrantProvision.sh

```diff
@@ -227,6 +239,7 @@ if ! ogrinfo --formats | grep --quiet FileGDB; then
     echo "### Building GDAL w/ FileGDB..."
     export PATH=/usr/local/lib:/usr/local/bin:$PATH
     cd gdal-1.10.1
+    touch config.rpath
```

Comment:
No there was a red warning about not finding the file.  Compilation succeeds either way, just removing an unnecessary warning.